### DISCUSSION TITLE
Linear Gradient as Button background color

### DISCUSSION THREAD

**Discussion by codakkk** [GeekyAnts/NativeBase#5184] [Feature request]
### Description

There's no way to pass Linear Gradient props. 

### Problem Statement

It's possible to pass Linear Gradient to every component except for Button.

### Proposed Solution or API

<Button bg={{linearGradient={....}}} />

### Alternatives

Trryied with parent Box: 
`
<Box
                bgColor={{
                  linearGradient: {
                    colors: ['#E31E24', '#B62322', '#962B24'],
                    start: [0, 0],
                    end: [1, 0],
                  },
                }}>
                <Button variant="unstyled">Applica Filtri</Button>
              </Box>
`
but with bad results (losing all the props on child, like rounded, etc)

I tryied to pass the bg to _stack with bad results.

**Comment by codakkk** [comment]
Lurking codebase, Pressable should be allowed to get Linear Gradient, not Button.

**Comment by mayank-96** [comment]
Hello @codakkk, we'll surely consider this API for `Pressable` in the next release. Thanks for pointing this out.

**Comment by johnf** [comment]
@mayank-96 let me know if you would be interested in a pull request and best approach
* Just copy the code from box int pressable
* Try and refactor Box and Pressable to use common code

**Comment by surajahmed** [comment]
@johnf Thanks for your suggestion. We'll make those changes in the coming release.

**Reply by caghp94** [reply]
@surajahmed has this been released?

**Comment by luluhoc** [comment]
@surajahmed Has this been implemented? How I can add a linear gradient to my buttons? Thanks
